src/script/common/c_packer.cpp

```diff
@@ -131,6 +132,7 @@ namespace {
 		}
 		T &operator*() { return (*vec)[idx]; }
 		T *operator->() { return &(*vec)[idx]; }
+		operator bool() { return vec != nullptr; }
```

Comment:
not that it matters but
```suggestion
		operator bool() const { return vec != nullptr; }
```